src/SFA.DAS.EmployerAccounts/Interfaces/IPensionRegulatorService.cs

```diff
@@ -6,8 +6,8 @@ namespace SFA.DAS.EmployerAccounts.Interfaces
 {
     public interface IPensionRegulatorService
     {
-        Task<IEnumerable<Organisation>> GetOrgansiationsByPayeRef(string payeRef);
-
-        Task<IEnumerable<Organisation>> GetOrgansiationsByAorn(string aorn, string payeRef);
+        Task<Organisation> GetOrganisationById(string payeRef);

```

Comment:
should the param name here be organisationId or something similar?

Reply:
Yep!